internal/usecase/devices/power.go

```diff
@@ -50,12 +50,8 @@ func (uc *UseCase) GetPowerState(c context.Context, guid string) (map[string]int
 
 func (uc *UseCase) GetPowerCapabilities(c context.Context, guid string) (map[string]interface{}, error) {
 	item, err := uc.GetByID(c, guid, "")
-	if err != nil {
-		return nil, nil
-	}
-
-	if item.GUID == "" {
-		return nil, nil
+	if err != nil || item.GUID == "" {
```

Comment:
What happens when we get a database error (failed to connect) --- would not found be the right thing to return?